src/Npgsql/NpgsqlDatabaseInfo.cs

```diff
@@ -97,19 +97,19 @@ public abstract class NpgsqlDatabaseInfo
 
         #region Types
 
-        List<PostgresBaseType>      _baseTypes      { get; } = new List<PostgresBaseType>();
-        List<PostgresArrayType>     _arrayTypes     { get; } = new List<PostgresArrayType>();
-        List<PostgresRangeType>     _rangeTypes     { get; } = new List<PostgresRangeType>();
-        List<PostgresEnumType>      _enumTypes      { get; } = new List<PostgresEnumType>();
-        List<PostgresCompositeType> _compositeTypes { get; } = new List<PostgresCompositeType>();
-        List<PostgresDomainType>    _domainTypes    { get; } = new List<PostgresDomainType>();
+        List<PostgresBaseType>      BaseTypesMutable      { get; } = new List<PostgresBaseType>();
+        List<PostgresArrayType>     ArrayTypesMutable     { get; } = new List<PostgresArrayType>();
+        List<PostgresRangeType>     RangeTypesMutable     { get; } = new List<PostgresRangeType>();
+        List<PostgresEnumType>      EnumTypesMutable      { get; } = new List<PostgresEnumType>();
+        List<PostgresCompositeType> CompositeTypesMutable { get; } = new List<PostgresCompositeType>();
+        List<PostgresDomainType>    DomainTypesMutable    { get; } = new List<PostgresDomainType>();
```

Comment:
It would be better to change these members to be `readonly` fields and not change names.

Reply:
Absolutely right.